addons/l10n_id_efaktur/models/efaktur.py

```diff
@@ -98,12 +98,20 @@ def _compute_default(self):
 
     @api.onchange('min')
     def _onchange_min(self):
-        self.min = '%013d' % int(re.sub(r'\D', '', self.min))
-        if not self.max or int(self.min) > int(self.max):
-            self.max = self.min
+        min_val = re.sub(r'\D', '', str(self.min))
+        if min_val:
+            self.min = '%013d' % int(min_val)
+            if not self.max or int(self.min) > int(self.max):
+                self.max = self.min
+        else:
+            raise ValidationError(_("Min must be numeric value!"))
```

Comment:
Don't forget to add changes in l10n_id_efaktur.pot

Reply:
It is useless in `l10n_` modules, they are not on Transpired.
Makes more noise than anything ;)